jaxrs/src/main/java/org/killbill/billing/jaxrs/resources/SubscriptionResource.java

```diff
@@ -357,6 +359,147 @@ public Response doResponseOk(final Entitlement entitlement) {
     }
 
     @TimedResource
+    @POST
+    @Path("/createEntitlementsWithAddOns")
+    @Consumes(APPLICATION_JSON)
+    @Produces(APPLICATION_JSON)
+    @ApiOperation(value = "Create multiple entitlements with addOn products")
+    @ApiResponses(value = {@ApiResponse(code = 400, message = "Invalid entitlements supplied")})
+    public Response createEntitlementsWithAddOns(final List<BulkBaseSubscriptionAndAddOnsJson> entitlementsWithAddOns,
```

Comment:
Could the previous call `createEntitlementWithAddOns` be implemented by calling this new one `createEntitlementSWithAddOns` by just passing a `List<BulkBaseSubscriptionAndAddOnsJson>` of size one, that is made from the input `final List<SubscriptionJson> entitlements` ?